src/plugin-slots/MobileUserMenuToggleSlot/README.md

```diff
@@ -0,0 +1,73 @@
+# Mobile User Menu Toggle Slot
+
+### Slot ID: `org.openedx.frontend.layout.header_mobile_user_menu_trigger.v1`
+
+## Description
+
+This slot is used to replace/modify/hide the contents of the user menu toggle button on mobile screens.
+
+## Examples
+
+### Modify Avatar
+
+The following `env.config.jsx` will modify the icon for the user menu toggle button on mobile.
+
+![Screenshot of modified items](./images/mobile_user_menu_toggle_modified_items.png)
+
+```jsx
+import { PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
+
+const modifyUserMenuToggle = ( widget ) => {
+  widget.content.avatar = "https://example.com/thumbs-up.png"
```

Comment:
Using `example.com` here goes against the "you can paste these examples into your own `env.config.jsx` and see what the screenshots show" strategy I've been pushing for with slot documentation. Any chance you could update this to use an image from a valid URL?

Reply:
@brian-smith-tcril I've updated this with a real-world dummy URL and updated the screenshots for consistency.